### DISCUSSION TITLE
Avatar, focus status for Avatar

### DISCUSSION THREAD

**Discussion by cdpestitschek** [twilio-labs/paste#2245] [Q&A] [answered]
**Background:** This iteration is about creating a foundation for User profile / settings / help center which we can start populating with useful information about the health status of users system, quick access settings and link to more preferences.
As the natural place (see Youtube, Gmail, Slack and also contact center competitors) for such content is user avatar in the header which we already have, the first step is to structure that area in a way that is future proof and can handle more items to be added.

**Agent Avatar**
With the statuses siting outside, the Avatar will be a place where we plan to incorporate the debugger so the agent can see their system status in a snapshot. In future iterations we will also add quick access audio device settings with option to go further to more comprehensive settings and start adding better information and instructions for any occurred issues.

**User statement**
As an agent, when I click on my avatar I expect to get visual feedback as confirmation that this area has been clicked.

*When the user clicks on the avatar a sidebar/modal will appear.

**Main question** 
> Do we need a focus status in Avatar? if yes, how it would look like?

**Need:** We would like clarification on focus status for Avatars
**Stage of design:** Define/High-fidelity
**Research:** TBD
**Links to mockups or prototype:** [Figma file](https://www.figma.com/proto/UtXzwC57mVVubxGPfxZmji/Help-Centre%2FSupportability?node-id=337%3A56194&scaling=min-zoom&page-id=142%3A29013&starting-point-node-id=337%3A56194&show-proto-sidebar=1)

![03 - Profile without name](https://user-images.githubusercontent.com/100429369/155743794-298bed67-d9b9-4f11-858a-2a6e97e65b17.png)

![Top Nav](https://user-images.githubusercontent.com/100429369/155743743-31fab06e-e623-49c2-b78d-6aa932fce183.png)

![Group 17](https://user-images.githubusercontent.com/100429369/155744346-2c82d4b6-430a-4127-9565-4d8e8b4e14f3.png)

**Comment by nkrantz** [comment, marked as answer]
Hi @cdpestitschek - if the component is interactive, it will need to be focusable. That said, it's not the actual Avatar component that will be clickable here, the Avatar will go inside of an interactive component (Menu or Button) in order for it to be used to trigger an action (open your help center). These components already have focus states so you don't have to worry about implementing the design of the focus border.

Feel free to check out the console masthead for an idea of what the focus state on the Avatar Menu should look like. Let us know if you have more questions!
